packages/eas-cli/src/utils/download.ts

```diff
@@ -162,16 +163,35 @@ export async function extractAppFromLocalArchiveAsync(
 }
 
 async function getAppPathAsync(outputDir: string, applicationExtension: string): Promise<string> {
-  const appFileName = await glob(`*.${applicationExtension}`, {
+  const appFilePaths = await glob(`./**/*.${applicationExtension}`, {
     cwd: outputDir,
     onlyFiles: false,
   });
 
-  if (appFileName.length === 0) {
-    throw Error('Something went wrong while extracting the app from app archive');
+  if (appFilePaths.length === 0) {
+    throw Error('Did not find any installable apps inside archive');
```

Comment:
Let me know what you think about the change:
```suggestion
    throw Error('Did not find any installable apps inside tarball');
```
In the past (and maybe even now), we would use the term "archive" to call an .apk, .aab, or .ipa file. If I'm not wrong, this function is used only after we extract a **tarball**. I think my suggestion makes the error message more clear.

Reply:
>I think my suggestion makes the error message more clear.

Yeah I think it will be more obvious if we use tarball instead of archive